openpype/hosts/blender/plugins/publish/collect_instances.py

```diff
@@ -114,10 +116,8 @@ def process(self, context):
                 }
             )
 
-            # Add instance holder as first item
-            members = list(members)
-
-            instance[:] = members
+            # Fill instance with members
+            instance[:] = list(members)
```

Comment:
Why not using `instance = list(members)` instead?

Reply:
Because `instance` is `class Instance(list)`, a list with custom attributes. If we do what you suggest, `instance` will become a basic `list` and loose all of these custom attributes we need.